server/health_handler.go

```diff
@@ -0,0 +1,8 @@
+package server
+
+import "net/http"
+
+// HealthHandler always reponds with 200 status
+func (s *HTTPServer) HealthHandler(w http.ResponseWriter, _ *http.Request) {
+	w.WriteHeader(http.StatusOK)
```

Comment:
Let’s add response body as well (like imgproxy [does](
https://github.com/DarthSim/imgproxy/blob/master/server.go#L222), for example, but without telling service’s name, i.e. “server is running”).

Not sure there is any particular reason for that; maybe, empty body might be considered as “bad response” by some clients/tools.

Reply:
@sponomarev Ready 😆